src/renderer/services/bitcoin/types.ts

```diff
@@ -39,7 +39,8 @@ export type FeesService = {
 }
 
 export type LedgerService = {
-  ledgerAddress$: LedgerAddressLD
-  retrieveLedgerAddress: () => void
-  removeLedgerAddress: () => void
+  ledgerMainnetAddress$: LedgerAddressLD
+  ledgerTestnetAddress$: LedgerAddressLD
```

Comment:
do we need them to be separated HERE ? 
lets do the same as to `retrieveLedgerAddress` and make it depends on the `network`